content/en/blog/falcoctl-gitlab-oci-artifacts-support.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "GitLab Container Registry Now Supports falcoctl OCI Artifact mediaTypes"
+linktitle: "GitLab Container Registry Now Supports falcoctl OCI Artifact mediaTypes"
+date: 2023-08-11
+author: Batuhan Apaydın
+slug: gitLab-container-registry-now-supports-falcoctl-oci-artifact-mediaTypes
+tags: ["Rules","Falco Plugins","Falcoctl", "GitLab", "OCI", "OCI Artifacts"]
+---
+
+Today, we are going to have a quick chit chat about the latest contribution we (w/[Emin Aktas](https://twitter.com/emminaktas)) made to GitLab's [Container Registry](https://gitlab.com/gitlab-org/container-registry).
+
+We noticed that GitLab Container Registry was not supporting falcoctl's OCI Artifact [mediaTypes]((https://github.com/falcosecurity/falcoctl/blob/7f1e8825a6f86010b9194577c56712dd0ef0442d/pkg/oci/constants.go#L20-L29) while we were pushing back the Falco's rules stored in GitHub's container registry to GitLab's container registry, so we decided to contribute to GitLab's Container Registry to add support for falcoctl's OCI Artifact mediaTypes.
```

Comment:
```suggestion
We noticed that GitLab Container Registry didn't support falcoctl OCI Artifact [mediaTypes](https://github.com/falcosecurity/falcoctl/blob/7f1e8825a6f86010b9194577c56712dd0ef0442d/pkg/oci/constants.go#L20-L29) while we were pushing the Falco rules stored from GitHub container registry to GitLab container registry. We decided then to contribute to GitLab Container Registry by adding the support for falcoctl OCI Artifact mediaTypes.
```

Reply:
thanks for fixing my bad grammar issues 🤦